schema/adminschema/v1.json

```diff
@@ -100,6 +100,37 @@
           "$ref": "Client"
         }
       }
+    },
+    "Connector": {
+      "id": "Connector",
+      "type": "any",
+      "description": "A configuration object which describes how dex will federate identity. Because the actual structure differs for each kind of connector, the object here is unspecified."
```

Comment:
you could be a bit more specific here - it's the JSON representation of stuff in connectors/connectors_XXX.go right?